src/views/BottomTabBar.js

```diff
@@ -220,9 +226,14 @@ class TabBarBottom extends React.Component<Props> {
         {routes.map((route, index) => {
           const focused = index === navigation.state.index;
           const scene = { route, focused };
-          const accessibilityLabel = this.props.getAccessibilityLabel({
-            route,
-          });
+          const accessibilityLabel = this.props.getAccessibilityLabel({ route });
+          const accessibilityRole = this.props.getAccessibilityRole({ route }) || 'button';
```

Comment:
What other accessibility roles instead of button do you have in mind?

Reply:
I think having `button` is a good default, but if someone only had an icon with no label, they may want to use `imagebutton`. Overriding `getAccessibilityRole` would allow that.

Reply:
The full list of available roles are here: https://facebook.github.io/react-native/docs/accessibility#accessibilityrole-ios-android